test/performance/observed_concurency_test.go

```diff
@@ -166,12 +165,6 @@ func testConcurrencyN(t *testing.T, concurrency int) []junit.TestCase {
 		t.Fatalf("Failed to create Service: %v", err)
 	}
 
-	t.Logf("%02d: waiting for deployment to scale to zero.", concurrency)
-	deploymentName := rsNames.Deployment(objs.Revision)
-	if err := e2e.WaitForScaleToZero(t, deploymentName, clients); err != nil {
```

Comment:
Where'd this go?

Reply:
Well that's part of the change.
If we're computing the latency only on the server side, then there is no reason to wait to scale to 0, since scale to 1 will always be "0.